<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TrafficLog</class>
 <widget class="QMainWindow" name="TrafficLog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>408</width>
    <height>222</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Traffic Log</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QTableWidget" name="tableWidget">
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="verticalScrollMode">
       <enum>QAbstractItemView::ScrollPerPixel</enum>
      </property>
      <property name="horizontalScrollMode">
       <enum>QAbstractItemView::ScrollPerPixel</enum>
      </property>
      <property name="columnCount">
       <number>3</number>
      </property>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>24</number>
      </attribute>
      <column/>
      <column/>
      <column/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::TopToolBarArea</set>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionEnableCapture"/>
   <addaction name="actionAutoScroll"/>
   <addaction name="actionClear"/>
  </widget>
  <action name="actionEnableCapture">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Enable Traffic Capturing</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="text">
    <string>Clear</string>
   </property>
  </action>
  <action name="actionAutoScroll">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Auto Scroll</string>
   </property>
  </action>
  <action name="actionTopmost">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Stay on Top</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
